moped-database/migrations/1698950636188_create_shared_use_trail_dual_path_component/up.sql

```diff
@@ -0,0 +1,62 @@
+INSERT INTO moped_components (
+    component_name,component_subtype,line_representation,feature_layer_id
+) VALUES
+    ('Trail', 'Shared Use Path (Paved Dual Trail)', TRUE, 4);
+
+-- insert all "protection type" subcomponents into moped_components_subcomponents
```

Comment:
this is a kinda fancy way of doing this that we probably want to book mark. it copies the subcomponent and work types of another similar component.

also, it does not rely on us knowing the `component_id` of the component we just created ☝️ instead we get that component ID by query on the component name + subtype.